src/NuGet.Core/NuGet.Packaging/Signing/Utility/AttributeUtility.cs

```diff
@@ -26,37 +26,34 @@ public static CryptographicAttributeObject CreateCommitmentTypeIndication(Signat
             // SignatureType -> Oid
             var oid = GetSignatureTypeOid(type);
 
-            var commitmentTypeQualifier = CommitmentTypeQualifier.Create(new Oid(oid));
-            var value = new AsnEncodedData(Oids.CommitmentTypeIndication, commitmentTypeQualifier.Encode());
+            var commitmentTypeIndication = CommitmentTypeIndication.Create(new Oid(oid));
+            var value = new AsnEncodedData(Oids.CommitmentTypeIndication, commitmentTypeIndication.Encode());
 
             return new CryptographicAttributeObject(
                 new Oid(Oids.CommitmentTypeIndication),
                 new AsnEncodedDataCollection(value));
         }
 
         /// <summary>
-        /// Gets the signature type from a commitment-type-indication attribute object.
+        /// Gets the signature type from one or more commitment-type-indication attributes.
         /// </summary>
-        /// <param name="attribute">A commitment-type-indication attribute object.</param>
+        /// <param name="signedAttributes">A <see cref="SignerInfo" /> signed attributes collection.</param>
         /// <remarks>Unknown OIDs are ignored.</remarks>
-        /// <exception cref="ArgumentNullException">Thrown if <paramref name="attribute" /> is <c>null</c>.</exception>
-        /// <exception cref="SignatureException">Thrown if <paramref name="attribute" /> is invalid.</exception>
-        public static SignatureType GetCommitmentTypeIndication(CryptographicAttributeObject attribute)
+        /// <exception cref="ArgumentNullException">Thrown if <paramref name="signedAttributes" /> is <c>null</c>.</exception>
+        /// <exception cref="SignatureException">Thrown if one or more attributes are invalid.</exception>
+        public static SignatureType GetSignatureType(CryptographicAttributeObjectCollection signedAttributes)
```

Comment:
@joelverhagen, [this](https://github.com/NuGet/NuGet.Jobs/pull/308/files#diff-14dd1b8313f77ac1145b58998d701d70R129) statement should change to:

```
var authorOrRepositoryCounterSignatureCount = packageSignature
    .SignerInfo
    .CounterSignerInfos
    .Cast<SignerInfo>()
    .Select(signerInfo => AttributeUtility.GetSignatureType(signerInfo.SignedAttributes))
    .Count(signatureType => signatureType != SignatureType.Unknown);
```

Reply:
Yup, that's why I have the catch `SignatureException`.